scripts/react_native_pods.rb

```diff
@@ -16,6 +16,11 @@ def use_react_native! (options={})
   # Include Hermes dependencies
   hermes_enabled = options[:hermes_enabled] ||= false
 
+  # Running `pod install` from inside Rosetta2 is a bad sign
+  if `/usr/sbin/sysctl -n hw.optional.arm64 2>&1`.to_i == 1 && !RUBY_PLATFORM.start_with?('arm64')
+    Pod::UI.warn 'Do not use Rosetta2 with "pod install"'
```

Comment:
what should people do instead? Maybe add a 2nd sentence pointing to a resource or list out the mitigation for them

Reply:
If there's no existing resource/doc about this, you can explain them in comment (in a paragraph form). And/or pull this out to a helper function so you can isolate the documentation.

Anyway your call :)